tests/union_duplicates.pyi

```diff
@@ -22,3 +23,17 @@ mixed_subscript_union: Union[str, Literal['foo'], typing_extensions.Literal['bar
 just_literals_pipe_union: TypeAlias = Literal[True] | Literal['idk']  # Y030 Multiple Literal members in a union. Use a single Literal, e.g. "Literal[True, 'idk']".
 mixed_pipe_union: TypeAlias = Union[Literal[966], int, Literal['baz']]  # Y030 Multiple Literal members in a union. Combine them into one, e.g. "Literal[966, 'baz']".
 many_literal_members_but_needs_combining: TypeAlias = int | Literal['a', 'b'] | Literal['baz']  # Y030 Multiple Literal members in a union. Combine them into one, e.g. "Literal['a', 'b', 'baz']".
+
+a: int | float  # Y041 "int" is redundant in a union with "float" (see PEP 484)
```

Comment:
- Currently the error message says what's wrong, not how to fix it. It might be easier to just tell how to fix the problem.
- PEP 484 is long, and only a couple paragraphs from it are related to this error. The current error message also doesn't help with finding the relevant part: searching for "redundant" finds only an unrelated part of the PEP.

How about an error message like this?

```
use "float" instead of "int | float" (see "The numeric tower" in PEP 484)
```

Reply:
Ah, that's much better, thanks.